api/api.js

```diff
@@ -0,0 +1,22 @@
+const apiURL = "https://pokeapi.co/api/v2/";
+
+export const getAllPokemon = async () => {
+  try {
+    const response = await axios.get(`${apiURL}pokemon/?limit=150&offset=0`);
```

Comment:
Quando usamos números assim para pegar dados da API, o ideal é colocar esses números (150, 0) numa variável, igual você fez com a `apiURL`, pra ficar mais fácil refatorar se for preciso.

Reply:
Li e lembrei de Leo falando hahaha vou ajeitar!